### PR TITLE
Homepage: Align about us & add GT in sponsors list

### PR DESCRIPTION
**Pull request by gautamjajoo**
Added GT to sponsors and aligned the question mark in about us.
@RishabhJain2018 

screencapture-127-0-0-1-6003-2019-05-30-00_41_14

### CODE REVIEW

**Comment by dhruvbatra** [comment]
Question mark is still too big in the screenshot.

**Comment by gautamjajoo** [comment]
@dhruvbatra I have updated the screenshot.

File: frontend/src/styles/partials/_home.scss [hunk 1]
```diff
@@ -399,3 +400,4 @@
       width: 100%;
       height: 100%;
     }
+
```
Comment (RishabhJain2018) [inline frontend/src/styles/partials/_home.scss]: Please add a new line here.

Comment (gautamjajoo) [reply frontend/src/styles/partials/_home.scss]: Done!

**Comment by RishabhJain2018** [comment]
Thanks for fixing it quickly @gautamjajoo! I'll merge it once the build completes.

**Pull request merged**